.travis.yml

```diff
@@ -34,6 +33,9 @@ jobs:
       name: "Run tests for Scala 2.12 and Java 11"
     - script: bin/test-documentation
       name: "Documentation validations and tests"
+    - script: bin/test-documentation
```

Comment:
On second thought. This approach is wrong: the `test-documentation` does more than _just_ `cd docs; sbt test`. It also validates markdown, invokes `unidoc` and checks formatting. 

I think for the time being `cd docs; sbt test` (using Java 11) should be enough.

Reply:
What's the concern with running everything it does on Java 11?